### PR TITLE
Address #1734. Add isTokenized method to MetadataHelper 

### PR DESCRIPTION
**Pull request by hlgp**
to support model validation.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
         <relativePath />
     </parent>
     <artifactId>metadata-utils</artifactId>
-    <version>1.11-SNAPSHOT</version>
+    <version>1.12-SNAPSHOT</version>
```
Comment (ivakegg) [inline pom.xml:11]: Why are you changing the snapshot version here?  This is done when we release.

File: src/main/java/datawave/query/util/AllFieldMetadataHelper.java [hunk 2]
```diff
@@ -152,7 +152,7 @@ public TypeMetadataHelper getTypeMetadataHelper() {
     }
     
     /**
-     * Method that fetches whether or not a column is indexed.
+     * Method that determines whether or not a column exists in the metadata table for the given key.
```
Comment (ivakegg) [inline src/main/java/datawave/query/util/AllFieldMetadataHelper.java:155]: wow... badly named method...

File: src/main/java/datawave/query/util/MetadataHelper.java [hunk 3]
```diff
@@ -570,6 +570,19 @@ public boolean isIndexed(String fieldName, Set<String> ingestTypeFilter) throws
         
     }
     
+    public boolean isTokenized(String fieldName, Set<String> ingestTypeFilter) throws TableNotFoundException {
```
Comment (ivakegg) [inline src/main/java/datawave/query/util/MetadataHelper.java:573]: method comment

**Review by ivakegg — CHANGES_REQUESTED** [review]

File: src/main/java/datawave/query/util/AllFieldMetadataHelper.java (hunk 2 above)
Comment (hlgp) [reply src/main/java/datawave/query/util/AllFieldMetadataHelper.java:155]: Indeed

File: pom.xml (hunk 1 above)
Comment (hlgp) [reply pom.xml:11]: will nuke

File: src/main/java/datawave/query/util/MetadataHelper.java (hunk 3 above)
Comment (hlgp) [reply src/main/java/datawave/query/util/MetadataHelper.java:573]: will do

File: src/main/java/datawave/query/util/AllFieldMetadataHelper.java (hunk 2 above)
Comment (hlgp) [reply src/main/java/datawave/query/util/AllFieldMetadataHelper.java:155]: want me to change it?

**Review by ivakegg — APPROVED** [review]

**Pull request merged**